ogcapi-stable/ogcapi-features-geojson/src/main/java/de/ii/ogcapi/features/geojson/domain/SchemaDeriverReturnables.java

```diff
@@ -37,7 +37,7 @@ public SchemaDeriverReturnables(
       String label,
       Optional<String> description,
       List<Codelist> codelists) {
-    super(version, schemaUri, label, description, codelists);
+    super(version, schemaUri, label, description, codelists, false);
```

Comment:
The value in the returnables is the key (unless there is a codelist transformation already in the provider, not "just" in Feature HTML). So, `false` needs to be `true` like for the queryables and sortables.

Reply:
The schema generation is based on constraints, it does not take into account value transformations. I would have assumed that a codelist constraint is accompanied by a codelist transformation, but that is obviously not the case in your example. 
We should further discuss this next week as part of the broader schema discussion I wanted to have.
I reverted the change for now.

Reply:
The codelist constraint in the feature schema only indicates that the value range should be from the list of codes in the codelist. Whether or where a codelist transformation is applied is another matter, typically only in the HTML representation. 
Agreed to discuss this in the broader schema context. We can mark this as resolved for this PR.